cogs/ReactForRole.py

```diff
@@ -217,6 +271,160 @@ async def rfr_edit_title(self, ctx: commands.Context):
         else:
             await ctx.send("Okay, cancelling command.")
 
+    @commands.check(KoalaBot.is_admin)
+    @commands.check(rfr_is_enabled)
+    @edit_group.command(name="thumbnail", aliases=["image", "picture"])
+    async def rfr_edit_thumbnail(self, ctx: commands.Context):
+        """
+        Edit the thumbnail of an existing rfr message. User is prompted for rfr message channel ID/name/mention, rfr
+        message ID/URL, new thumbnail, Y/N confirmation. User needs admin perms
+        :param ctx: Context of the command
+        :return:
+        """
+        await ctx.send("Okay, this will edit the thumbnail of a react for role message. I'll need some details first "
+                       "though.")
+        msg, channel = await self.get_rfr_message_from_prompts(ctx)
+        embed = self.get_embed_from_message(msg)
+        if not embed:
+            KoalaBot.logger.error(
+                f"RFR: Can't find embed for message id {msg.id}, channel {channel.id}, guild id {ctx.guild.id}.")
+        await ctx.send(f"Your current image here is {embed.thumbnail.url}")
+        image = await self.prompt_for_input(ctx, "image you'd like to use as a thumbnail")
+        if not image or image == "":
+            await ctx.send("Okay, cancelling command.")
+        if isinstance(image, discord.Attachment):
+            # correct type
```

Comment:
The type of the attachment should also be checked - we only want to accept images for thumbnails

This can be done through checking the [content_type](https://discordpy.readthedocs.io/en/latest/api.html?highlight=attachment#discord.Attachment.content_type) which is new to discord 1.7 - Koalabot has been updated to use discord 1.7 so this should be available for you to use.